Lacrit/assignment4/Islands.h

```diff
@@ -0,0 +1,15 @@
+#include <vector>
+
+class Islands 
+{
+	private: 
+		std::vector<std::vector<bool>> grid; 
+	public: 
+		int rows;
+		int cols;
+		Islands(std::vector<std::vector<bool>> grid);
+		bool isValid(const std::vector<std::vector<bool>> &mat, int row, int col, int rows, int cols, const std::vector<std::vector<bool>> &visited);
```

Comment:
`isValid()` & `DFS()` could also be private. They are not useful for a user, right? They are part of the implementation of `countIslands()`